tests/util/gentle-register-property.js

```diff
@@ -6,15 +6,16 @@
  * @param {string} [meta.syntax] - Property syntax.
  * @param {boolean} [meta.inherits] - Whether the property inherits.
  * @param {*} [meta.initialValue] - Initial value.
+ * @param {Window} [window] - The window to check in.
  * @returns {boolean} - Whether the property was successfully registered.
  */
-export default function gentleRegisterProperty (property, meta = {}, global = globalThis) {
-	if (!property.startsWith("--") || !global.CSS?.registerProperty) {
+export default function gentleRegisterProperty (property, meta = {}, window = globalThis) {
+	if (!property.startsWith("--") || !window.CSS?.registerProperty) {
```

Comment:
I don't fully understand why we need to rename this parameter. Could you please elaborate?

But we definitely need to add it to the JSDocs. Somehow, we've missed it. 😅

Reply:
Oh, I see. Because we have another name in the corresponding util function. Well, in that case, that makes sense.